docs/developer/attributes.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Attributes
+sidebar_label: Attributes
+---
+
+## Introduction
+
+Setting up attributes will allow your users to filter your products catalogue.
+
+Attributes can be used for
+- [Products](developer/api-reference/objects/product.mdx) and [ProductVariants](developer/api-reference/objects/product-variant.mdx)
+- [Pages](developer/api-reference/objects/page.mdx)
+
+## Use cases
+
+1. Your store sells Mugs in many colors and sizes. This means you will need:
```

Comment:
Your store sells mugs in many colors and sizes. In this example, you can set up the following attributes: